### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by RasmusOrsoe**
Adds a getting started guide.

@asogaard should have fixed indentation

### CODE REVIEW

File: getting_started.md [hunk 1]
```diff
@@ -0,0 +1,767 @@
...
+
+
+## 4. The `Dataset` and `DataLoader` classes
+
+The `Dataset` class in GraphNeT is based on [`torch.utils.data.Dataset`](https://pytorch.org/docs/stable/data.html#torch.utils.data.Dataset)s.
+This class is responsible for reading data from file and preparing them as a graph-object, for which we use the  [`torch_geometric.data.Data`](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.data.Data.html#torch_geometric.data.Data) class.
+The `Dataset` class currently comes in two flavours;
+
+- [ParquetDataset](https://graphnet-team.github.io/graphnet/api/graphnet.data.parquet.parquet_dataset.html): Lets you prepare raphs based on data read from Parquet files.
+ - [SQLiteDataset](https://graphnet-team.github.io/graphnet/api/graphnet.data.sqlite.sqlite_dataset.html): Lets you prepare graphs based on data read from SQLite databases.
```
Comment (asogaard) [inline getting_started.md:77]:
```suggestion
- [SQLiteDataset](https://graphnet-team.github.io/graphnet/api/graphnet.data.sqlite.sqlite_dataset.html): Lets you prepare graphs based on data read from SQLite databases.
```

File: getting_started.md [hunk 2]
```diff
@@ -0,0 +1,767 @@
...
+```python
+from graphnet.data.sqlite import SQLiteDataset
+
+dataset = SQLiteDataset(
+    path="data/examples/sqlite/prometheus/prometheus-events.db",
+	pulsemaps="total",
+	truth_table="mc_truth",
+	features=["sensor_pos_x", "sensor_pos_y", "sensor_pos_z", "t", ...],
+	truth=["injection_energy", "injection_zenith", ...],
+)
```
Comment (asogaard) [inline getting_started.md:98]:
```suggestion
from graphnet.data.sqlite import SQLiteDataset

dataset = SQLiteDataset(
    path="data/examples/sqlite/prometheus/prometheus-events.db",
    pulsemaps="total",
    truth_table="mc_truth",
    features=["sensor_pos_x", "sensor_pos_y", "sensor_pos_z", "t", ...],
    truth=["injection_energy", "injection_zenith", ...],
)
```

File: getting_started.md [hunk 3]
```diff
@@ -0,0 +1,767 @@
...
+```python
+from graphnet.data.parquet import ParquetDataset
+
+dataset = ParquetDataset(
+    path="data/examples/parquet/prometheus/prometheus-events.parquet",
+    pulsemaps="total",
+	truth_table="mc_truth",
+	features=["sensor_pos_x", "sensor_pos_y", "sensor_pos_z", "t", ...],
+	truth=["injection_energy", "injection_zenith", ...],
+)
```
Comment (asogaard) [inline getting_started.md:114]:
```suggestion
from graphnet.data.parquet import ParquetDataset

dataset = ParquetDataset(
    path="data/examples/parquet/prometheus/prometheus-events.parquet",
    pulsemaps="total",
    truth_table="mc_truth",
    features=["sensor_pos_x", "sensor_pos_y", "sensor_pos_z", "t", ...],
    truth=["injection_energy", "injection_zenith", ...],
)
```

File: getting_started.md [hunk 4]
```diff
@@ -0,0 +1,767 @@
...
+detector = Prometheus(graph_builder=graph_builder)
+gnn = DynEdge(
+    nb_inputs=detector.nb_outputs,
+    global_pooling_schemes=["min", "max", "mean"],
+)
+task = ZenithReconstructionWithKappa(
+    hidden_size=gnn.nb_outputs,
+	target_labels="injection_zenith",
+	loss_function=VonMisesFisher2DLoss(),
+)
```
Comment (asogaard) [inline getting_started.md:395]:
```suggestion
task = ZenithReconstructionWithKappa(
    hidden_size=gnn.nb_outputs,
    target_labels="injection_zenith",
    loss_function=VonMisesFisher2DLoss(),
)
```

File: getting_started.md [hunk 5]
```diff
@@ -0,0 +1,767 @@
...
+index_column = "event_no"
+truth_table = "mc_truth"
+event_no = 120
+
+with sqlite3.connect(database) as conn:
+	query = f"SELECT * from {pulsemap} WHERE {index_column} == {event_no}"
+	detector_response = pd.read_sql(query, conn)
+
+	query = f"SELECT * from {truth_table} WHERE {index_column} == {event_no}"
+	truth = pd.read_sql(query, conn)
```
Comment (asogaard) [inline getting_started.md:766]:
```suggestion
with sqlite3.connect(database) as conn:
    query = f"SELECT * from {pulsemap} WHERE {index_column} == {event_no}"
    detector_response = pd.read_sql(query, conn)

    query = f"SELECT * from {truth_table} WHERE {index_column} == {event_no}"
    truth = pd.read_sql(query, conn)
```

File: getting_started.md
Comment (asogaard) [inline getting_started.md:1]: To make the naming consistent with other Markdown files in the root directory, I think the file name should be all upper-case, i.e., `GETTING_STARTED.md`, alternatively `TUTORIAL.md` to be consistent with the title in the document.

**Review by asogaard — APPROVED** [review]
Hi @RasmusOrsoe,

I have added one comment regarding the file name, and a few suggestions regarding indentation, but I think this is ready to merge. :)

**Comment by RasmusOrsoe** [comment]
@asogaard done.

**Pull request merged**